nixos/modules/security/pam.nix

```diff
@@ -253,6 +267,8 @@ let
               "auth sufficient ${pkgs.pam_u2f}/lib/security/pam_u2f.so"}
           ${optionalString cfg.usbAuth
               "auth sufficient ${pkgs.pam_usb}/lib/security/pam_usb.so"}
+          ${optionalString cfg.googleAuth
+              "auth required ${pkgs.googleAuthenticator}/lib/security/pam_google_authenticator.so"}
```

Comment:
> Also, does the global enableGoogleAuth option make GA required for console logins? I think by default most users would not want to have 2FA required for local logins, only remote ones.

Hmm, it would. I can't seem to find any pam file specifically for local, as opposed to network logins...

I could enable OTP only for SSH, but I'd prefer a blacklist rather than a whitelist. Anyone who enables OTP is security-conscious, and probably don't want random new services to bypass it. Not sure how to do that, though...

> In the 3rd case, I think it's strange to ask for OTP first, then the password; most users will not be accustomed to this and security systems should be familiar to prevent mistakes (e.g. typing password into Google authenticator while on autopilot!). I think it's better to make this 2FA instead of 2SV to mitigate the "figure-out-the-password-without-OTP" vuln, by asking for the password, then always asking for OTP, then accepting/rejecting based on the combination.

I'm not sure how to do this.